packages/workers-shared/asset-worker/src/index.ts

```diff
@@ -73,25 +74,25 @@ export default class extends WorkerEntrypoint<Env> {
 				this.ctx,
 				this.env.SENTRY_DSN,
 				this.env.SENTRY_ACCESS_CLIENT_ID,
-				this.env.SENTRY_ACCESS_CLIENT_SECRET
+				this.env.SENTRY_ACCESS_CLIENT_SECRET,
+				this.env.COLO_METADATA,
+				this.env.CONFIG.account_id,
+				this.env.CONFIG.script_id,
 			);
 
 			const config = applyConfigurationDefaults(this.env.CONFIG);
 			const userAgent = request.headers.get("user-agent") ?? "UA UNKNOWN";
 
-			if (sentry) {
-				const colo = this.env.COLO_METADATA.coloId;
-				sentry.setTag("colo", this.env.COLO_METADATA.coloId);
-				sentry.setTag("metal", this.env.COLO_METADATA.metalId);
-				sentry.setUser({ userAgent: userAgent, colo: colo });
-			}
-
 			const url = new URL(request.url);
 			if (this.env.COLO_METADATA && this.env.VERSION_METADATA) {
 				analytics.setData({
+					accountId: this.env.CONFIG.account_id,
+					scriptId: this.env.CONFIG.script_id,
+
 					coloId: this.env.COLO_METADATA.coloId,
 					metalId: this.env.COLO_METADATA.metalId,
 					coloTier: this.env.COLO_METADATA.coloTier,
+
```

Comment:
?

Reply:
Just separating out the sections (indexes, doubles and blobs).